src/main/java/uk/gov/hmcts/reform/unspec/service/docmosis/sealedclaim/SealedClaimFormGenerator.java

```diff
@@ -65,6 +67,7 @@ private String getFileName(CaseData caseData) {
 
     @Override
     public SealedClaimForm getTemplateData(CaseData caseData) {
+        Optional<SolicitorReferences> solicitorReferences = Optional.ofNullable(caseData.getSolicitorReferences());
```

Comment:
is this an issue in confirmation of service too?

Reply:
Good catch. yes it is broken there as well. fixing

Reply:
fixed